### ISSUE TITLE
[Redesign] Add test files for “Import EARL as JSON” to wiki

### ISSUE THREAD

**Issue by hidde** [w3c/wcag-em-report-tool#478]
@nitedog is asking around to get test files from various vendors so that we can:

* test the import functionality
* find out what the minimum requirements are for an import file to work

**Comment by hidde** [comment]
@nitedog did you receive any test files?

**Comment by nitedog** [comment]
No, just referrals to these implementation reports in EARL/JSON:
 - https://github.com/act-rules

The assertions are likely about test rules rather than success criteria but they can probably be used to create test files with a little bit of search & replace.

**Comment by hidde** [comment]
So in terms of this issue… maybe we close it for now, as we don't and probably won't have minimal requirements or working test files?
